hypervisor/lazy.go

```diff
@@ -49,6 +49,13 @@ func statePreparing(ctx *VmContext, ev VmEvent) {
 		ctx.setWindowSize(cmd.ClientTag, cmd.Size)
 	case COMMAND_RUN_POD, COMMAND_REPLACE_POD:
 		glog.Info("got spec, prepare devices")
+		err := ctx.DCtx.(LazyDriverContext).InitVM(ctx)
```

Comment:
You picked initVM from LazyLaunch, right? LazyVmLoop is called before prepare devices. Moving initVM to lazyVmLoop  makes more sense.